### PR TITLE
Add orientation tracking for Roboy's hands

### PR DESCRIPTION
**Pull request by rasaford**
Closes XSF-4

- [x] Add checkbox to turn orientation tracking on- and off

### CODE REVIEW

**Review by missxa — CHANGES_REQUESTED** [review]
exclude files not directly relevant for the changes, like XR Rig.prefab and the settings files

File: Assets/Roboy/Wheelchair/Scripts/OrientationOffset.cs [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    void Start()
+    {
+        
+    }
+
+    // Update is called once per frame
+    void Update()
+    {   
+        // Derived by experimentation
+        Quaternion offset = Quaternion.Euler(-189.118f, -8.403992f, 15.2381f);
```
[the remark this answers is not in the document]
Comment (rasaford) [reply Assets/Roboy/Wheelchair/Scripts/OrientationOffset.cs:19]: Aparrently const Quartenion is not allowed, but I defined the rotations to be const

**Comment by rasaford** [comment]
- [ ] Add checkbox to turn orientation tracking on- and off

**Comment by rasaford** [comment]
Added a track offset checkbox, which enables / disables orienation tracking on the OrientationOffsets

Capture

File: Assets/Roboy/Wheelchair/Scripts/OrientationOffset.cs [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+        // Derived by placing an empty GameObject at the writs position and
+        // having the IK goal following it's rotation. 
+        // By then applying rotations manually to this game object to visually align Roboy's
+        // hand with the Oculus controllers we derived the following constants
+        const float rx = -189.118f, ry = -8.403992f, rz = 15.2381f;
```
Comment (missxa) [inline Assets/Roboy/Wheelchair/Scripts/OrientationOffset.cs:26]: no need to declare this in every update. these values never change, so they can be moved to start() or just static private members

File: Assets/Roboy/Wheelchair/Scripts/OrientationOffset.cs [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+        Quaternion offset = Quaternion.Euler(rx, ry, rz);
+
+        if (trackOrientation)
+        {
+            orientationObjective.Weight = 1;
```
Comment (missxa) [inline Assets/Roboy/Wheelchair/Scripts/OrientationOffset.cs:31]: where is this variable used?

**Review by missxa — CHANGES_REQUESTED** [review]

**Review by missxa — APPROVED** [review]

**Pull request merged**